model/src/main/kotlin/Vulnerability.kt

```diff
@@ -43,4 +43,37 @@ data class Vulnerability(
      */
     @JsonInclude(JsonInclude.Include.NON_NULL)
     val url: URL? = null
-)
+) {
+    /**
+     * See https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale.
```

Comment:
I'd like to see a short description what this rating is about like you described it in your commit message.

Reply:
Done. I've removed the sentence from the commit message in exchange.